src/Core/Services/Sentinel/Sentinel.Api/wwwroot/css/account/login.css

```
@@ -0,0 +1,211 @@
+﻿:root {
```

Biome (2.1.2)

[error] Unknown type selector is not allowed.

See MDN web docs for more details.
Consider replacing the unknown type selector with valid one.

(lint/correctness/noUnknownTypeSelector)